i3pystatus/core/desktop.py

```diff
@@ -55,4 +66,12 @@ def display(self):
             if self.timeout:
                 notification.set_timeout(self.timeout)
             notification.set_urgency(self.URGENCY_LUT[self.urgency])
-            return notification.show()
+            try:
+                return notification.show()
+            except Exception as exc:
```

Comment:
I just realized that I'm not doing anything with this exception, so assigning it to ``exc`` here is unnecessary. Do you want me to remove this?